tests/test_end_to_end/test_utils.py

```diff
@@ -0,0 +1,20 @@
+from openapi_python_client import utils
```

Comment:
I try to make the tests file structure mirror the module file structure to make finding tests a bit easier.  The exception being `test_end_to_end` which tests more than one piece of code. So could you move this test file up a directory?